scripts/install/authelia.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+#
+if [[ ! -f /install/.nginx.lock ]]; then
+    echo_warn "Nginx is required for this application"
+    exit
+fi
+
+. /etc/swizzin/sources/functions/utils
+. /etc/swizzin/sources/functions/os
+
+# Get the current version using a git ls-remote tag check
+authelia_latestv="$(git ls-remote -t --refs https://github.com/authelia/authelia.git | awk '{sub("refs/tags/", "");sub("(.*)-alpha(.*)", ""); print $2 }' | awk '!/^$/' | sort -rV | head -n1)"
+# Create the download url using the version provided by authelia_latestv
+case "$(_os_arch)" in
+    "amd64") authelia_arch="amd64" ;;
+    "arm32") authelia_arch="arm32v7" ;;
+    "arm64") authelia_arch="arm64v8" ;;
+    *)
+        echo_error "Arch not supported"
+        exit 1
+        ;;
+esac
+
+authelia_url="https://github.com/authelia/authelia/releases/download/${authelia_latestv}/authelia-linux-${authelia_arch}.tar.gz"
+# Create the loction for the stored binary
+mkdir -p "/opt/authelia"
+# Download the binary
+wget -qO "/opt/authelia/authelia-linux-amd64.tar.gz" "${authelia_url}"
+# Extract the specific file we need and nothing else.
+tar -xf "/opt/authelia/authelia-linux-amd64.tar.gz" -C "/opt/authelia/" 'authelia-linux-amd64'
+# Symlink the extracted binary authelia-linux-amd64 to authelia
+ln -fsn "/opt/authelia/authelia-linux-amd64" "/opt/authelia/authelia"
```

Comment:
curious, why symlink this? I mean, it will work, just wondering why tho

Reply:
because they decided to include the arch in the binary name. authelia-linux-ARCH. Since I cannot rename while extracting it's less complex to just create a symlink instead trying to rm, mv or cp, rm.